frontend/src/metabase/entities/data-apps/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { DataApp } from "metabase-types/api";
+
+export function getDataAppIcon(app: DataApp) {
+  return { name: "star" };
+}
```

Comment:
It's not so much about customization (yet), but because we'd need this helper for the collection entity. Collections have their own `getIcon` function and we need to extend it, so it uses `getDataAppIcon` if a collection is a data app